src/LoRa.h

```diff
@@ -81,7 +81,8 @@ class LoRaClass : public Stream {
   // deprecated
   void crc() { enableCrc(); }
   void noCrc() { disableCrc(); }
-
+  
+  void LoRandomSeed();
```

Comment:
The new API needs to be added to `keywords.txt` and `API.md`. Note, I've suggested an alternative name above.